core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -14,42 +14,38 @@
 
 /**
  * @fileoverview Page object for the contributor dashboard admin page,
- * for use in Protractor tests.
+ * for use in Webdriverio tests.
  */
 
 var action = require('./action.js');
 var waitFor = require('./waitFor.js');
 
 var ContributorDashboardAdminPage = function() {
-  var CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION = 'TRANSLATION';
+  var CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION = 'REVIEW_TRANSLATION';
```

Comment:
For selecting option using the visible text we need to provide the actual string present in the `<option>string<option>`, passing partial string does't work. So I added the actual string present in the options instead of partial one.